worlds/pokemon_emerald/__init__.py

```diff
@@ -706,6 +706,7 @@ def fill_slot_data(self) -> Dict[str, Any]:
             "allowed_legendary_hunt_encounters",
             "extra_bumpy_slope",
             "free_fly_location",
+            "free_fly_blacklist",
```

Comment:
Does this need to be in slot data?

Reply:
I don't think it does, thinking about it, so I've removed it